acj/static/modules/common/avatar-directive.js

```diff
@@ -6,12 +6,13 @@
 			scope: {
 				userId: '=',
 				avatar: '=',
+                forceDefault: '=?',
 				displayName: '=?',
 				fullName: '=?',
 				me: '=?'
 			},
 			template: '<a ng-href="#/user/{{ userId }}">' +
-						'<img src="//www.gravatar.com/avatar/{{ avatar }}?s=32&d=retro" /> ' +
+						'<img src="//www.gravatar.com/avatar/{{ avatar }}?s=32&d=retro{{ forceDefault ? \'&f=y\' : \'\' }}" /> ' +
```

Comment:
If we are using the same image for everyone, there is no reason to have an image at all. The image is used to differentiate one author from another when scanning answers and replies. I definitely am in favour of keeping the avatars distinct.